Source/Core/DolphinQt/Debugger/CodeDiffDialog.cpp

```diff
@@ -418,52 +449,48 @@ void CodeDiffDialog::OnGoTop()
 
 void CodeDiffDialog::OnDelete()
 {
-  // Delete from include and listwidget.
-  int remove_item = m_matching_results_list->row(m_matching_results_list->currentItem());
-  if (!remove_item || remove_item == -1)
+  // Delete from include list and qtable widget
+  auto item = m_matching_results_table->currentItem();
+  if (!item)
+    return;
+  int row = m_matching_results_table->row(item);
+  if (row == -1)
     return;
-  m_include.erase(m_include.begin() + remove_item - 1);
-  m_matching_results_list->takeItem(remove_item);
+  // TODO: If/when sorting is ever added, .erase needs to find item position instead; leaving as is
+  // for performance
+  m_include.erase(m_include.begin() + row);
+  m_matching_results_table->removeRow(row);
 }
 
 void CodeDiffDialog::OnSetBLR()
 {
-  auto item = m_matching_results_list->currentItem();
+  auto item = m_matching_results_table->currentItem();
   if (!item)
     return;
+
   Common::Symbol* symbol = g_symbolDB.GetSymbolFromAddr(item->data(Qt::UserRole).toUInt());
   if (!symbol)
     return;
   PowerPC::debug_interface.SetPatch(symbol->address, 0x4E800020);
-  item->setForeground(QBrush(Qt::red));
+
+  int row = item->row();
+  m_matching_results_table->item(row, 0)->setForeground(QBrush(Qt::red));
+  m_matching_results_table->item(row, 1)->setForeground(QBrush(Qt::red));
+  m_matching_results_table->item(row, 2)->setForeground(QBrush(Qt::red));
+  m_matching_results_table->item(row, 3)->setForeground(QBrush(Qt::red));
+  m_matching_results_table->item(row, 3)->setText(QStringLiteral("X"));
+
   m_code_widget->Update();
 }
 
 void CodeDiffDialog::UpdateItem()
 {
-  auto item = m_matching_results_list->currentItem();
-  auto address = item->data(Qt::UserRole).toUInt();
-  auto fullstring = item->text().toStdString();
-  auto symbolName = g_symbolDB.GetDescription(address);
-  int row = m_matching_results_list->row(item);
+  QTableWidgetItem* item = m_matching_results_table->currentItem();
+  int row = m_matching_results_table->row(item);
   if (!row || row == -1)

```

Comment:
Need to change this otherwise first item will never be updated.

Reply:
Waited for pipeline.
Pipelines successfully passed, 

rebase to latest master and fixing this mistake.